src/PaymentGateways/PaypalSettingPage.php

```diff
@@ -0,0 +1,114 @@
+<?php
+namespace  Give\PaymentGateways;
+
+use Give\PaymentGateways;
+use function give_get_current_setting_section as getCurrentSettingSection;
+
+/**
+ * Class PaypalSettingSection
+ * @package Give\PaymentGateways
+ *
+ * @sicne 2.8.0
+ */
+class PaypalSettingPage implements SettingPage {
+	/**
+	 * @var PayPalCheckout\PayPalCheckout
+	 */
+	private $paypalCheckout;
+
+	/**
+	 * @var PayPalStandard\PayPalStandard
+	 */
+	private $paypalStandard;
+
+	/**
+	 * Register properties
+	 * @return PaypalSettingPage
+	 *
+	 * @since 2.8.0
+	 */
+	public function register() {
+		$this->paypalCheckout = new PayPalCheckout\PayPalCheckout();
+		$this->paypalStandard = new PaymentGateways\PayPalStandard\PayPalStandard();
+
+		return $this;
+	}
```

Comment:
This can be replaced with a constructor and then use [auto-injection](https://laravel.com/docs/7.x/container#automatic-injection) with the service container to pull them in.

Reply:
fixed: https://github.com/impress-org/givewp/compare/8f8e9bd44851...014ff89da09f